### PR TITLE
Update docker image for database to pgrouting + set DB port in .env

### PR DESCRIPTION
**Pull request by nicolas-heigvd**
Update docker image for the database (`db` service in docker-compose) to [pgrouting](https://hub.docker.com/layers/pgrouting/pgrouting/12-3.0-2.6.3/images/sha256-b8681c6ebb78c6e681ab03c54b92d9b41ec4bc224c881e2301a88bc7cd9d098b?context=explore) + set DB port as an environment variable in .env.sample (default port is probably already used on your local machine).

Need reviewing before merge.

### CODE REVIEW

**Comment by nicolas-heigvd** [comment]
@verdonarthur Could you also check if this fix your issues please?

File: packages/backend/.env.sample [hunk 1]
```diff
@@ -6,6 +6,7 @@ COMPOSE_PROJECT_NAME=modos_backend
 MODOS_BASE_URL=http://localhost:3000
 MODOS_DATABASE_URL=postgres://modos@localhost/modos
 MODOS_PORT=3000
+MODOS_DB_PORT=5432
```
Comment (verdonarthur) [inline packages/backend/.env.sample:9]: Not necessary as it is not used in the application

File: packages/backend/docker-compose.yml [hunk 2]
```diff
@@ -28,13 +28,13 @@ services:
...
   db:
-    image: postgres:12.3-alpine
+    image: pgrouting/pgrouting:12-3.0-2.6.3
     environment:
       POSTGRES_DB: modos
       POSTGRES_USER: modos
       POSTGRES_PASSWORD: modos
     ports:
-      - "5432:5432"
+      - "${MODOS_DB_PORT}:5432"
```
Comment (verdonarthur) [inline packages/backend/docker-compose.yml:37]: I don't think it's necessary to have something dynamic here

**Comment by verdonarthur** [comment]
env.sample is for the application configuration, not for the docker compose. If you want to have a env file for your docker-compose, you need to add a new one (https://docs.docker.com/compose/environment-variables/)

**Comment by nicolas-heigvd** [comment]
Should be ok now.

Don't forget to:
`cp .env.docker.sample .env.docker` before running the docker-compose.yml.

README.me was modified accordingly.

**Review by verdonarthur — APPROVED** [review]

**Pull request merged**